### PR TITLE
Fixed the comment adding

### PR DESCRIPTION
**Pull request by holus-bolus**

### CODE REVIEW

**Review by Wiret02 — APPROVED** [review]

**Review by iva-stasia — APPROVED** [review]

**Review by iaremchuk — APPROVED** [review]

**Review by VictoriaRodik — APPROVED** [review]

**Comment by holus-bolus** [comment]
The code for the services/comments has been changed. The addComments function has been overwritten
image


We descturcturize the response from the user and get the according fields. then we look for this user in the database. At last we create a comment.





the comment schema has been changed as well
image


instead of just adding the author we get the user type from the scema.

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,10 @@
 MIT License
 
+<<<<<<< HEAD
 Copyright (c) 2021 ITA-Social-Projects
+=======
+Copyright (c) 2023 ITA-Social-Projects
+>>>>>>> 6e233d92b9ccda0646f6abfbf6806d082da02b31
```
Comment (vlad-khrychov) [inline LICENSE]: resolve conflict

File: README.md [hunk 2]
```diff
@@ -1,3 +1,323 @@
+<<<<<<< HEAD
 # SpaceToStudy-BackEnd
 
 [![Build Status](https://dev.azure.com/ProjectApproach/Space2Study/_apis/build/status/ita-social-projects.SpaceToStudy-BackEnd?branchName=develop)](https://dev.azure.com/ProjectApproach/Space2Study/_build/latest?definitionId=32&branchName=develop)
+=======
```
Comment (vlad-khrychov) [inline README.md]: Also conflict

File: utils/cooperations/coopsAggregateOptions.js [hunk 3]
```diff
@@ -15,6 +15,8 @@ const coopsAggregateOptions = (params = {}, query) => {
   } else {
     sortOption[parsedSort.orderBy] = sortOrder
   }
+  console.log('Query parameters:', query)
+  console.log('User Params:', params)
```
Comment (vlad-khrychov) [inline utils/cooperations/coopsAggregateOptions.js]: Delete `console.log`-s

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]
So, now do we save in DB duplicated information? I think you should change not post endpoint but get endpoint where we obtain comments. In this case from DB you will obtain author id, then using it obtain info about author, then merge all necessary info in response body and after that send response to client.

Also in this PR a lot of additional changes (LICENSE, README). Delete them from this PR. If you want update project documentation it must be separate PR.

**Comment by holus-bolus** [comment]
@vlad-khrychov I corrected the code, but I can't reopen the pull request. It says that it has no history with the develop and main

**Comment by vlad-khrychov** [comment]
> @vlad-khrychov I corrected the code, but I can't reopen the pull request. It says that it has no history with the develop and main

@holus-bolus  open a new PR with your current changes.

**Comment by holus-bolus** [comment]
> > @vlad-khrychov I corrected the code, but I can't reopen the pull request. It says that it has no history with the develop and main
> 
> @holus-bolus open a new PR with your current changes.
@vlad-khrychov , it's closed and doesn't allow me to open a PR to the closed issue

**Pull request closed without merging**